src/query.ts

```diff
@@ -26,7 +26,7 @@ type RecordCollectionCallback<TFields extends FieldSet> = (
     error: CallbackError,
     records?: Records<TFields>
 ) => void;
-type DoneCallback = (error: CallbackError) => void;
+type DoneCallback = (error: CallbackError, other?: any) => void;
```

Comment:
Curious why do we need this change here?

Reply:
This was throwing a ts error when used in a ts project.